Week-4/Exercise4.3/Exercise4.3.js

```diff
@@ -0,0 +1,25 @@
+let i = 2;
```

Comment:
Although the below solution is correct but still calling next() 10 times is kind of repetitive. It could had better if could keep this logic inside a wrapper function and then calling that function with limit argument.
eg - 
const Fib = (n) => ({
    [Symbol.iterator]: () => {
        let i = 1, prev = 0, current = 0;
        return {
            next: () => {
                if (i++ <= n) {
                    [prev, current] = [current, (prev + current) || 1];
                    return {
                        value: prev,
                        done: false
                    }
                }
                else {
                    return {
                        done: true
                    }
                }
            }
        }
    }
});

console.log([...Fib(8)]);